### PR TITLE
HiFi Hamburger Menu pt. 2

### PR DESCRIPTION
**Pull request by thejefflu**
CHANGED:
- implemented hi-fi designs
- added log out and close drawer buttons

TODO:
- airtable functionality (wait for profile screen to be completed?)
- route "past orders" and "log out" options to proper screens (once their respective screens are completed)

### CODE REVIEW

File: src/Navigation/CustomDrawer.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+          style={{
+            marginLeft: 'auto', marginRight: 10, marginTop: 10,
+          }}
+          onPress={() => { navigation.toggleDrawer(); }}
+        />
+        <View style={{ marginLeft: 30, marginTop: 30 }}>
+          <Image
+            // eslint-disable-next-line global-require
+            source={require('../assets/imgs/placeholder.png')}
+            style={{
```
Comment (hwang635) [inline src/Navigation/CustomDrawer.js]: move long styles to stylesheet

File: src/Navigation/CustomDrawer.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+          }}
+        >
+          <Icon
+            size={26}
+            name="logout-variant"
+            color="#34221D"
+            onPress={() => { navigation.navigate('SignIn'); }} // TODO: change to signed out screen once it is implemented
+          />
+          <Text
+            style={{
```
Comment (hwang635) [inline src/Navigation/CustomDrawer.js]: move long styles to stylesheet

File: src/Navigation/CustomDrawer.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+import React from 'react';
+import { DrawerContentScrollView, DrawerItemList } from '@react-navigation/drawer';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+import PropTypes from 'prop-types';
+
+function CustomDrawer(props) {
+  const { navigation } = props;
+
+  return (
+    <View style={{
```
Comment (hwang635) [inline src/Navigation/CustomDrawer.js]: change the fonts please!

**Review by hwang635 — CHANGES_REQUESTED** [review]
Hi Jeff! This looks really nice. Only things to do seem to be the fonts and stylesheet. I can't run this on my own emulator but I'm going to come back to it later to see if there's a serious issue.

**Comment by curesh** [comment]
Note that code only worked after following these two stackoverflows.
- https://stackoverflow.com/questions/67130651/reanimated-2-failed-to-create-a-worklet-maybe-you-forgot-to-add-reanimateds-ba
- https://stackoverflow.com/questions/46878638/how-to-clear-react-native-cache

**Comment by curesh** [comment]
Also, log out button throws an error. Ideally we want it to just not work (be greyed out), since we havent implementing logging out functionality yet.

**Pull request merged**